test/testUtils/ufeUtils.py

```diff
@@ -55,6 +55,26 @@ def selectPath(path, replace=False):
     else:
         ufeSelectCmd.append(sceneItem)
 
+def makeUfePath(dagPath, sdfPath=None):
```

Comment:
Quite oddly named, I would say...  I would have expected makeUfePath to return a Ufe.Path, not a scene item!

Reply:
Right. Any suggestions for the name?

Reply:
createUfeSceneItem(), createUfeItem(), makeUfeSceneItem(), makeUfeItem().  Take your pick!

Reply:
Done - https://github.com/Autodesk/maya-usd/pull/838/commits/0dd8d8a59e58f46af60ecb49b4091b50ec0a64fb